docs-source/content/userguide/istio/istio.md

```diff
@@ -188,22 +232,23 @@ using HTTP on port 80, and a virtual service that will route all of
 those requests to the cluster service for `cluster-1` in `domain1` in
 the namespace `domain1`.
 
+Once the gateway and virtual service has been setup, you can access it through your ingress host and port, 
+refer to the section `Determining the ingress IP and ports` in [Istio Getting Started](https://istio.io/latest/docs/setup/getting-started/)
+
```

Comment:
Use the specific link: https://istio.io/latest/docs/setup/getting-started/#determining-the-ingress-ip-and-ports

Reply:
Fixed